actors.md

```diff
@@ -4,6 +4,42 @@ Any implementations of the Filecoin actors must be exactly byte for byte compati
 
 This spec decsribes a set of actors that operate within the [Filecoin State Machine](state-machine.md). All types are defined in [the basic type encoding spec](data-structures.md#basic-type-encodings).
 
+## Init Actor
+The init actor is responsible for creating new actors on the filecoin network. This is a built-in actor and cannot be replicated. In the future, this actor will be responsible for loading new code into the system (for user programmable actors).
+
+```go
+type InitActor struct {
+    NextActorID uint64
+}
+```
+
+#### Exec
+
+Parameters:
+
+- code Cid
```

Comment:
where do I get this Cid from?

Reply:
We should have a list of well known CIDs somewhere for the builtin actors